src/cuttlefish_escript.erl

```diff
@@ -230,6 +232,43 @@ describe(ParsedArgs, [Query|_]) when is_list(Query) ->
     end,
     stop_deactivate().
 
+%% This is the function that dumps the value for a single setting.
+%% To be shell friendly, returns nothing when something is wrong.
+get(_ParsedArgs, []) ->
```

Comment:
this is a special version of describe.
while I think it makes sense to add a new command name,
but I'd suggest to move the common parts to a common function which is called by both `describe` and `get`.

i.e. the function collects all the info to print for `describe` command,
but `get` command only prints the set value.

Reply:
fixed

Reply:
ok.
i was thinking maybe 
get(Args, Queries) -> desc(Args, Queries, get).
describe(Args, Queries) -> desc(Args, Queries, describe).

then `desc` is can print only the value when the third arg is `get`

Reply:
`describe` sometimes prints stuff to stdout, while `get` only prints them to debug log. My idea is to keep "print" parts away from commonization.

Reply:
ok